packages/bot-web-ui/src/components/summary/summary-card.tsx

```diff
@@ -75,6 +75,7 @@ const SummaryCard = observer(({ contract_info, is_contract_loading }: TSummaryCa
                 'db-summary-card--completed': is_contract_completed,
                 'db-summary-card--completed-mobile': is_contract_completed && is_mobile,
             })}
+            data-testid='mock-summary-card'
```

Comment:
@maryia-matskevich-deriv I think we need to use `dt_` prefix before the `data-testid`. @vinu-deriv previously suggested me to follow this pattern.